deploy-ecs/deploy.sh

```diff
@@ -61,17 +61,28 @@ if [ "${REQUIRES_SECRETS}" = true ] && (echo "${newcontainers}" | jq '.[0] | .se
 fi
 
 echo "Publishing new task definition."
```

Comment:
Maybe adding the launch type to this?
```suggestion
echo "Publishing new ${LAUNCH_TYPE} task definition."
```